lib/server.js

```diff
@@ -131,31 +145,82 @@ async function startHttpsServer(config) {
         cert: keys.certificate,
       };
     }
-    await https.createServer(options, app(config)).listen(port, host);
-    logger.info(`[${process.pid}] HTTPS: listening on port ${port}`);
+
+    return new Promise((resolve, reject) => {
+      const srv = https.createServer(options, app(config)).listen(port, host, (err) => {
```

Comment:
btw: I think that `await https.createServer(app(config)).listen(port, host)` never worked.... https.createServer is not promisified.